src/Tests/Sending/When_routing_outgoingmessages_to_endpoints.cs

```diff
@@ -676,6 +678,176 @@ public Task Handle(BrokeredMessage brokeredMessage)
                 throw new MessageTooLargeException();
             }
         }
-    }
 
+        [Test]
+        public async Task Should_route_via_active_namespace_first_no_ASB_required()
```

Comment:
Do we need `_no_ASB_required` in the name of the test. The same goes for the other one.